src/main/java/uk/gov/digital/ho/hocs/info/api/TeamService.java

```diff
@@ -152,12 +153,12 @@ public Team createTeam(TeamDto newTeam, UUID unitUUID) {
             unit.addTeam(team);
         } else {
             log.debug("Team {} exists, not creating.", newTeam.getDisplayName());
-            createKeyCloakMappings(team.getUuid(), Optional.empty());
+            keycloakService.createTeamGroupIfNotExists(team.getUuid());
             throw new ApplicationExceptions.EntityAlreadyExistsException(
```

Comment:
This constructor for `EntityAlreadyExistsException` causes an `UNCAUGHT_EXCEPTION` in Kibana. Any chance you could add a log event that is more representative? Would help with log searches in Kibana...

Reply:
I have addressed this